Sources/Public/CalendarViewRepresentable.swift

```diff
@@ -447,12 +449,55 @@ extension CalendarViewRepresentable {
 @available(iOS 13.0, *)
 extension CalendarViewRepresentable {
 
+  /// Configures the day-selection handler.
+  ///
+  /// It is the responsibility of your feature code to decide what to do with each day. For example, you might store the most recent day
+  /// in a selected day property, then read that property in your `dayItemProvider` closure to add specific "selected" styling to a
+  /// particular day view. If one of your item provider closures depends on this selected day state, remember to include it as part of the
+  /// `dataDependency` parameter when initializing your `CalendarViewRepresentable`.
+  ///
+  /// - Parameters:
+  ///   - daySelectionHandler: A closure (that is retained) that is invoked whenever a day is selected.
   public func onDaySelection(_ daySelectionHandler: @escaping (Day) -> Void) -> Self {
     var view = self
     view.daySelectionHandler = daySelectionHandler
     return view
   }
 
+  /// Configures the multiple-day-selection drag handler.
+  ///
+  /// Multiple selection is initiated with a long press, followed by a drag / pan. As the gesture crosses over more days in the calendar,
+  /// this handler will be invoked with each new day. It is the responsibility of your feature code to decide what to do with this stream of
+  /// days. For example, you might convert them to `Date` instances, and use them as input to the `dayRangeItemProvider`. If
+  /// one of your item provider closures depends on state referencing this stream of selected days, remember to include it as part of
+  /// the `dataDependency` parameter when initializing your `CalendarViewRepresentable`.
+  ///
+  /// - Parameters:
+  ///   - began: A closure (that is retained) that is invoked when the multiple-day-selection drag gesture begins.
+  ///   - changed: A closure (that is retained) that is invoked when the multiple-day-selection drag gesture intersects a new day.
+  ///   - ended: A closure (that is retained) that is invoked when the multiple-day-selection drag gesture ends.
+  public func onMultipleDaySelectionDrag(
+    began: @escaping (Day) -> Void,
+    changed: @escaping (Day) -> Void,
+    ended: @escaping (Day) -> Void)
```

Comment:
we can keep the same naming as `DragGesture` in SwiftUI
```suggestion
    onEnded: @escaping (Day) -> Void)
```

Reply:
oh nice, updating!

Reply:
although... it doesn't really read as well to repeat the word `on` (which is already at the beginning of the function call). Thoughts? Maybe it's ok to slightly deviate here since this API isn't 1:1 with how SwiftUI gestures work anyways.

Reply:
`onMultipleDaySelectionDrag(began:changed:ended:)` reads more like a grammatically-correct english sentence than `onMultipleDaySelectionDrag(onBegan:onChanged:onEnded:)`, right?

Reply:
ah good point, makes sense to leave as is